salt/modules/file.py

```diff
@@ -1693,6 +1693,13 @@ def line(path, content=None, match=None, mode=None, location=None,
 
             salt '*' file.line /path/to/file content="CREATEMAIL_SPOOL=no" match="CREATE_MAIL_SPOOL=yes" mode="replace"
     '''
+    def _split(content):
+        split_content = []
+        for linesep_line in content.split(os.linesep):
+            for content_line in linesep_line.split('\n'):
```

Comment:
This code makes a difference only being executed on platforms with os.linesep != '\n'... Why not use str.splitlines() instead?

Reply:
I'm aware of that.
That specific part of code was tested in file.blockreplace.